src/components/shared/inputs/FormattedNumberInput.tsx

```diff
@@ -30,10 +30,6 @@ export default function FormattedNumberInput({
   accessory?: JSX.Element
   onChange?: (val?: string) => void
 } & FormItemExt) {
-  const [accessoryWidth, setAccessoryWidth] = useState<number>(0)
```

Comment:
Not sure why this was necessary, but it suddenly started causing issues.

I think absolute position is a totally valid solution here instead.